src/PhotometryModel.cc

```diff
@@ -0,0 +1,38 @@
+#include "lsst/log/Log.h"
+
+#include "lsst/jointcal/CcdImage.h"
+#include "lsst/jointcal/PhotometryModel.h"
+
+namespace lsst {
+namespace jointcal {
+
+bool PhotometryModel::validate(CcdImageList const &ccdImageList) const {
+    bool check = true;
+    for (auto const &ccdImage : ccdImageList) {
+        check &= checkPositiveOnBBox(*ccdImage);
+    }
```

Comment:
Consider short-circuiting here, it would probably be both faster and more readable.

Reply:
As with the internal loop, I want to log every invalid case, so I don't want to short circuit. Comment added.